packages/paste-core/components/form/src/FormTextArea.tsx

```diff
@@ -2,59 +2,57 @@ import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import {styled, css} from '@twilio-paste/styling-library';
 import TextareaAutosize from 'react-autosize-textarea';
-import {FieldWrapper} from './shared/FieldWrapper';
-import {Prefix} from './shared/Prefix';
-import {Suffix} from './shared/Suffix';
+import {FormControlWrapper} from './shared/FormControlWrapper';
 import {safelySpreadFormControlProps} from './shared/Utils';
 
 export interface FormTextAreaProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
+  className?: never;
```

Comment:
Are you manually sorting these or did you get a VSCode plugin to do it?

Reply:
@SiTaggart filled me in on this plugin: https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines